PushNotifications/PushNotifications/PushNotifications.h

```diff
@@ -1,5 +1,7 @@
 #import <Foundation/Foundation.h>
-
+#if TARGET_OS_IPHONE
+#import <UserNotifications/UserNotifications.h>
```

Comment:
This is needed for iOS projects using Objective-C if you specify custom `UNAuthorizationOptions` to avoid the following error: `Declaration of 'UNAuthorizationOptionAlert' must be imported from module 'UserNotifications.UNUserNotificationCenter' before it is required`.